geth/configEmpty.toml

```diff
@@ -0,0 +1,13 @@
+[Eth]
+NetworkId = 8995
```

Comment:
maybe we should ask Fabian about `NetworkId` before merging to the `master` branch?